src/providers/pact-publisher-provider.test.ts

```diff
@@ -1,14 +1,14 @@
 import { Test } from '@nestjs/testing';
-import { Publisher } from '@pact-foundation/pact';
+import { Publisher } from '@pact-foundation/pact-core';
 import { PactModuleProviders } from '../common/pact-module-providers.enum';
-import { PactPublisherProvider } from '../providers/pact-publisher.provider';
+import { PactPublisherProvider } from './pact-publisher.provider';
 
-jest.mock('@pact-foundation/pact');
+const mockPublisher = jest.createMockFromModule<Publisher>('@pact-foundation/pact-core');
```

Comment:
I have some troubles here understanding how this works. Seems to work now, but not 100% sure.